documentation/modules/adding-source-provider.adoc

```diff
@@ -26,37 +26,166 @@ You can add {a-rhv} source provider by using the {ocp} web console.
 
 * {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate
 endif::[]
+ifdef::ova[]
+= Adding an Open Virtual Appliance (OVA) source provider
+
+You can add Open Virtual Appliance (OVA) files that were created by or are compatible with VMware vSphere as source providers.
+
+[IMPORTANT]
+====
+Migration using Open Virtual Application (OVA) files that were created by or are compatible with VMware vSphere as source providers is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during the development process.
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+endif::[]
+ifdef::ostack[]
+= Adding an {osp} source provider
+
+You can add an {osp} source provider by using the {ocp} web console.
+
+[NOTE]
+====
+Migration using {osp} source providers only supports VMs that use only Cinder volumes.
+====
+endif::[]
+ifdef::cnv[]
+= Adding a Red Hat {virt} source provider
+
+You can add a Red Hat {virt} source provider by using the {ocp} web console.
+
+endif::[]
+ifdef::cnv2[]
+= Adding {a-virt} destination provider
+
+You can add another {virt} destination provider to the {ocp} web console in addition to the default {virt} destination provider, which is the provider where you installed {project-short}.
+
+.Prerequisites
+
+* You must have {a-virt} link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/authentication_and_authorization/using-service-accounts[service account token] with `cluster-admin` privileges.
+endif::[]
 
 .Procedure
 
 . In the {ocp} web console, click *Migration* -> *Providers for virtualization*.
 . Click *Create Provider*.
 ifdef::vmware[]
-. Select *VMware* from the *Provider type* list.
+. Click *vSphere*.
```

Comment:
interestingly, we got a request to say vSphere rather than VMware in the documentation because he got the wrong impression that we support imports from ESX. so that's good!

Reply:
:)